code/datums/mutations/telekinesis.dm

```diff
@@ -7,6 +7,10 @@
 	text_gain_indication = "<span class='notice'>You feel smarter!</span>"
 	limb_req = BODY_ZONE_HEAD
 	instability = 30
+	///Typecache of atoms that TK shouldn't interact with
+	var/list/blacklisted_atoms = list(
+		/atom/movable/screen,
+	)
```

Comment:
- Static list?
- Pedantic but this isn't actually a typecache, but a list of types